tox.ini

```diff
@@ -7,5 +7,6 @@ deps =
     mock
     sh
     pyyaml: PyYAML>=3.10
+    cov4: coverage>=4.0a4
```

Comment:
I guess we need to be explicit and force coverage 3.x for another env to be future compatible when coverage 4.x will become default.

Maybe like that
```
{py26,py27,py32,py33,py34,pypy,pypy3}-{cov3,cov4 {py26,py27,py32,py33,py34,pypy,pypy3}-pyyaml
```